src/sass/partials/components/_forms.scss

```diff
@@ -0,0 +1,78 @@
+
+// Content Forms
+.content-form,
+.users-form {
+  padding: 1em 0;
+
+  label {
+    display: block;
+    margin: 5px 0;
+  }
+
+  input {
+    display: block;
+    padding: 5px;
+  }
+
+  .form--field {
+    display: grid;
+    grid-template-columns: auto;
+    grid-template-rows: auto;
+    padding: .5em 0;
+  }
+
+  .form--alert {
+    color: red;
+    font-size: .8em;
+  }
+
+  .form--description {
+    font-size: .8em;
+    padding: 5px 0;
+  }
+  @include breakpoint($small) {
+    label,
+    input,
+    .form-description {
+      display: block;
+    }
+  }
+
+  fieldset {
+    margin: 1em 0;
+  }
+}
+
+
+
+// Users Table
+@email-width: 200px;
```

Comment:
I'm not sure what this is…
Sass declares variables with $, so `$email-width`. This looks like Less to me.